compute_sdk/globus_compute_sdk/sdk/client.py

```diff
@@ -294,7 +294,13 @@ def get_batch_result(self, task_id_list):
         return results
 
     @requires_login
-    def run(self, *args, endpoint_id=None, function_id=None, **kwargs) -> str:
+    def run(
+        self,
+        *args,
+        endpoint_id: UUID_LIKE_T = None,
+        function_id: UUID_LIKE_T = None,
+        **kwargs,
```

Comment:
Probably worth either removing `**kwargs` or adding a warning in line with `__init__`